static/styles/left_sidebar.css

```diff
@@ -591,6 +684,26 @@ li.expanded_private_message {
     }
 
     .zoom-in-hide {
-        display: none;
+        display: none !important;
```

Comment:
oh here's one more `!important`!

Reply:
I noticed this `!important` when you mentioned about the previous one, but I think we would necessarily need this one as we're trying to hide this `div` when we are in `more private messages` view.

Attaching screenshot without `!important` on this class - 

<img width="274" alt="image" src="https://user-images.githubusercontent.com/76561593/182774328-083223ef-e8fc-4174-b998-ddfd9a96ca95.png">

Reply:
I see why you need `display: none;`, but why doesn't it work if you don't add `!important`? What other style is taking over instead? Usually there's a way to make this style work without having to add `!important`

Reply:
@evykassirer , here's the `display:flex` which is the reason behind this - 
<img width="303" alt="image" src="https://user-images.githubusercontent.com/76561593/182776578-d1f93029-69a9-41d6-86b2-487614fe77dc.png">

Reply:
You can do 

`    
    #default_private_messages_view_header.zoom-in-hide {
        display: none;
    }
`

The extra `#default_private_messages_view_header` is a way of telling it that you're deliberately overwriting the other style for `#default_private_messages_view_header` with a more specific situation.

Reply:
Umm, this too doesn't seems to work

Reply:
and particularly, I don't think we should have much problem with `!important` present here, as even looking into our codebase, there are several places where we have used `!important` for getting `display: none` done.

Reply:
@evykassirer , I made this work by -- 

`
#default_private_messages_view_header {
        display: none;
}
`

Reply:
writing `#default_private_messages_view_header .zoom-in-hide` isn't right.

Reply:
Thanks!

Reply:
you need to do it with no space in between!

Reply:
the space makes it so that it expects the second selector to be on an element _inside_ the first one, whereas with no space, it shows that you're describing the _same_ div but with more detail. Try that?